low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -163,8 +168,56 @@ def get_cpu_info(self):
         ]
         return cpu_data
 
+    def format_ipmi_platform_sensor_reading(self, reading):
+        """builds json resposne from ipmi tool response.
+        reading arg sample: ('CPU1 Temp', '01', 'ok', '3.1', '36 degrees C')
+        """
+        uid = '_'.join(reading[0].split())
+        sensor_id = reading[0]
+        sensor_props = self._ipmi.get_sensor_props(sensor_id)
+        lower_critical = sensor_props[1].get('Lower Critical', 'NA')
+        upper_critical = sensor_props[1].get('Upper Critical', 'NA')
+        lower_non_recoverable = sensor_props[1].get('Lower Non-Recoverable', 'NA')
+        upper_non_recoverable = sensor_props[1].get('Upper Non-Recoverable', 'NA')
+        status = 'Ok' if reading[2] == 'ok' else 'NA'
```

Comment:
It should be `OK` (instead of `Ok`) just to be consistent with other PRs.

Reply:
Done.

Reply:
Wait, i asked him to make it 'Ok' thinking thats how its used everywhere